Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM elixir:latest
```

Comment:
don't do this. 
When you use `latest`, docker will always use latest elixir version available. We should be aware of the version change as at times, things can break when migrating to a new version. 
Use otp alpine based image from https://hub.docker.com/r/hexpm/elixir/tags